src/components/KTruncate/KTruncate.vue

```diff
@@ -134,14 +136,15 @@ const kTruncateContainer = ref<HTMLDivElement>()
 const kTruncateWrapper = ref<HTMLDivElement>()
 const textToggleControls = ref<HTMLDivElement>()
 const wrapperHeight = ref<string>('0px')
+const gap = KUI_SPACE_40
```

Comment:
question: why do we need to set this to a `gap` variable instead of just referencing the `KUI_SPACE_40` variable directly?

Reply:
Because we reference it twice - in case if that value changes at some point, so we don't forget about the other occurrence

Reply:
I'm not sure I understand. Why can't you just reference the space const?

Reply:
Added a comment there. Basically this gap is used in one of the functions to perform some calculations. But it's also used in styles. So it's important, if we ever change one we use in styles, to not forget to update it in script too. If it's set to a local const - it's only defined in one place.

Reply:
Does that make sense?

Reply:
you're talking about here? Why can't you reference in both?
```
gap: v-bind('KUI_SPACE_40');
```

Reply:
Oh sorry - yes I can (don't even have to use JS const - I can use SCSS one instead) but what I was trying to do is to not allow this value be changed in styles but not in script. Basically if tomorrow we decide we want a bigger gap there a developer might come it and change it to:
```
gap: v-bind('KUI_SPACE_50');
```

However the change in script will remain untouched and that will break the component. If it's defined in one place, they will change the definition:
```
const gap = KUI_SPACE_50
```

But both usages in style and in script will reference the same value.

Reply:
Ah ok that makes more sense. Can you leave a comment above the gap variable explaining?

Reply:
Done